### ISSUE TITLE
[Request] Completely Disable Personalized Suggestions

### ISSUE THREAD

**Issue by Z3ROGRAV1TYx** [AnySoftKeyboard/AnySoftKeyboard#1810]
**Can we please get the option to completely disable personalized input suggestions? (Without "removing suggestions").**

What I mean is, similar to in GBoard, you can disable personalized suggestions but keep suggestions enabled. 
Example: If I type 'l8r' often, it shows up on the suggestion strip. So my proposal is to be able to disable this completely, but keep it so as your typing, it still shows suggestions based off common words. So if I type 'lat', then "Later" will show up on that suggestion strip. You just won't be seeing that _PERSONALIZED_ 'l8r' for example.

I hope that was clear enough, sorry if it was confusing.

**Comment by samantohermes** [comment]
Disable user-dictionary and next word suggestions

**Comment by Z3ROGRAV1TYx** [comment]
@samantohermes yes but I want to keep next word suggestoons on. (This is possible on gboard). It looks at what your typing and can suggest, but doesnt personalize any results in the strip.

**Comment by samantohermes** [comment]
So disable instead user-dictionary and auto dictionary

**Comment by ArenaL5** [comment]
The options are there, but indeed they're broken. Thanks for your report, @ Z3ROGRAV1TYx.

There are workarounds to disable word learning for now:

1. Open AnySoftKeyboard from your app drawer, and when you see the welcome screen, click the Language button below (the icon is a globe).
2. Click Configure correction behavior and other dictionaries.
3. **This option may be broken:**

[] *User-dictionary*
Suggest user added words (double-click a typed suggestion to edit)

So if that's not working, continue this guide.
4. Scroll down, and look for this option:

*Auto dictionary*
Automatically learn new words.

Ideally, you should be able to set it to "Do not learn new words", but that option was only repaired on AnySoftKeyboard Dev. If you're not willing to install the developers version, set it to "After some usage", so the keyboard will only learn a word after 10 uses.

5. Scroll up again, click Word editor, and remove your custom words. Repeat this step periodically if "Do not learn new words" doesn't work for you.
